debian/libcart-dev.install

```diff
@@ -0,0 +1,4 @@
+usr/include/cart/*
```

Comment:
should a name of this be libract1-dev? or does major version only get added to non-dev packages?

Reply:
Generally the major version gets added to the -dev package only if it is not compatible with the older -dev package and both need to be installed on the same system.

Reply:
Is that even a thing.  Versioned -dev packages?  I'm not sure I've ever come across one.

@frostedcmos But yes, it's typical for the library package to have the major (ABI brokenness portion of the libraries version) portion of the version appended to the library package.  This allows for multiple ABI versions of the library to be installed to be compatible with the other packages that might want old while others want the new ABI.  Probably preaching to the choir here.